test/python/test_const.py

```
@@ -0,0 +1,87 @@
+from __future__ import print_function
+
+import pytest
+
+import tensorflow as tf
+import os
+
+from common import NgraphTest
+
+# Uncomment for debugging; Also add -s in command like, e.g.
+# (venv-tf-py3) [build_cmake]$
+# NGRAPH_TF_LOG_PLACEMENT=1 NGRAPH_TF_VLOG_LEVEL=6 pytest -s -k test_const_scalarval ../test/python/test_const.py
+import logging
+logging.basicConfig(level=logging.DEBUG)
+
+
+class TestConstOperations(NgraphTest):
+
+    def test_const_listvals(self):
+        zz = tf.constant([1, 2, 3, 4, 5, 6], dtype=float, shape=[2, 3])
+
+        def run_test(sess):
+            return sess.run(zz)
+
+        assert (
+            self.with_ngraph(run_test) == self.without_ngraph(run_test)).all()
+
+    def test_const_listvals_2(self):
+        zz = tf.constant([[1, 2, 3], [4, 5, 6]], dtype=float, shape=[2, 3])
+
+        def run_test(sess):
+            return sess.run(zz)
+
+        assert (
+            self.with_ngraph(run_test) == self.without_ngraph(run_test)).all()
+
+    def test_const_scalarval(self):
+        zz = tf.constant(-3, dtype=float, shape=[2, 3])
+
+        def run_test(sess):
+            return sess.run(zz)
+
+        assert (
+            self.with_ngraph(run_test) == self.without_ngraph(run_test)).all()
+
+    def test_const_lastfill(self):
+        zz = tf.constant([1, 2], dtype=float, shape=[2, 3])
+
+        def run_test(sess):
+            return sess.run(zz)
+
+        assert (
+            self.with_ngraph(run_test) == self.without_ngraph(run_test)).all()
+
+    # This test will not pass as NGraph will throw error, unlike TF which fills in zeros
+    @pytest.mark.skip(
+        reason="NGraph will throw error, but TF will fill in zeros")
+    def test_const_empty(self):
+        log = logging.getLogger('test_const_empty')
+        zz = tf.constant([], dtype=float, shape=[2, 3])
+
+        def run_test(sess):
+            log.debug('Invoking sess.run(zz)')
+            return sess.run(zz)
+
+        assert (
```

instead of skipping the test we could assert on error being thrown?